.github/workflows/codeql-analysis.yml

```diff
@@ -78,3 +78,21 @@ jobs:
        ./build_Vic2ToHoI4_release.sh
     - name: Perform CodeQL Analysis
       uses: github/codeql-action/analyze@v1
+      
+    - name: Get Release by Tag
+        id: get_release_by_tag
+        uses: jonfriesen/get-release-by-tag@v1.0.0
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          tag_name: 0.2Nimitz
```

Comment:
Aren't you on Oppenheimer now?

Reply:
Ah, yes, but no tag yet. Gonna delay attempting to auto-push Linux builds.